src/murmur/Server.h

```diff
@@ -169,6 +169,11 @@ class Server : public QThread {
 		// Certificate stuff, implemented partially in Cert.cpp
 	public:
 		static bool isKeyForCert(const QSslKey &key, const QSslCertificate &cert);
+		/// Attempt to load a private key in PEM format from |buf|.
+		/// If |passphrase| is non-empty, it will be used in trying to decipher the key.
```

Comment:
Shouldn't it be `/// If |passphrase| is non-empty, it will be used when trying to decipher the key.` or `/// If |passphrase| is non-empty, it will be used to try to decipher the key.`?